git/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id("kotlin")
+    id("java-test-fixtures")
+}
+
+val kotlinVersion: String by project
+val mockitoJunit5Version: String by project
+
+dependencies {
+    implementation(gradleApi())
+    implementation(project(":kotlin-dsl-support"))
+    // can't use logging due to cyclic dependency
+    implementation(project(":utils"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
```

Comment:
Can we use this property directly from gradle.properties? Is it available implicitly?
I'd like to omit "val <propertyName>: String by project" if possible

Reply:
afaik no